### PR TITLE
Add community updates to README

### PR DESCRIPTION
**Pull request by vincentschen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,17 @@ If you want to set up a development environment in order to contribute back to S
...
-We tag questions with "Q&A", and you can see them [here](https://github.com/snorkel-team/snorkel/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Q%26A%22+).
+
+## Community
+### Spectrum
+For broader Q&A, discussions about using Snorkel, tutorial requests, etc., we've launched a community forum: https://spectrum.chat/snorkel
+
+We hope this will be a venue for you to interact with other Snorkel users — please don't be shy about posting!
+
+### Google Groups
+If you'd like to stay up-to-date about the latest Snorkel-related announcements, releases, and workshops, please sign up for the mailing list on Google Groups: https://groups.google.com/forum/#!forum/snorkel-ml
```
Comment (bhancock8) [inline README.md]: mention this will be sparse.

File: README.md [hunk 2]
```diff
@@ -31,8 +31,17 @@ If you want to set up a development environment in order to contribute back to S
...
-We use [GitHub Issues](https://github.com/snorkel-team/snorkel/issues) as a place to put bugs, questions, feature requests, and other stuff.
-Don't be shy about posting!
+## Issues and feature requests
+We use [GitHub Issues](https://github.com/snorkel-team/snorkel/issues) as a place to put bugs and feature requests — anything code-related.
 Just make sure you search for related issues first and use our Issues templates.
-We tag questions with "Q&A", and you can see them [here](https://github.com/snorkel-team/snorkel/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Q%26A%22+).
+
+## Community
+### Spectrum
+For broader Q&A, discussions about using Snorkel, tutorial requests, etc., we've launched a community forum: https://spectrum.chat/snorkel
```
Comment (bhancock8) [inline README.md]: Drop 'we've launched' and just say 'we have' since this README will be around for a while.

File: README.md [hunk 3]
```diff
@@ -31,8 +31,17 @@ If you want to set up a development environment in order to contribute back to S
 see our [Contributing doc](./CONTRIBUTING.md).
 
 
-## Issues and Q & A
-We use [GitHub Issues](https://github.com/snorkel-team/snorkel/issues) as a place to put bugs, questions, feature requests, and other stuff.
-Don't be shy about posting!
+## Issues and feature requests
+We use [GitHub Issues](https://github.com/snorkel-team/snorkel/issues) as a place to put bugs and feature requests — anything code-related.
```
Comment (bhancock8) [inline README.md]: put -> post

File: README.md [hunk 4]
```diff
@@ -31,8 +31,17 @@ If you want to set up a development environment in order to contribute back to S
...
 Just make sure you search for related issues first and use our Issues templates.
-We tag questions with "Q&A", and you can see them [here](https://github.com/snorkel-team/snorkel/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Q%26A%22+).
+
+## Community
+### Spectrum
+For broader Q&A, discussions about using Snorkel, tutorial requests, etc., we've launched a community forum: https://spectrum.chat/snorkel
+
+We hope this will be a venue for you to interact with other Snorkel users — please don't be shy about posting!
+
+### Google Groups
```
Comment (bhancock8) [inline README.md]: Group

File: README.md (hunk 1 above)
Comment (bhancock8) [reply README.md]: e.g,. "To stay up-to-date on only the biggest Snorkel-related announcements (e.g., version releases, upcoming workshops, etc.), join the Google Group mailing list:
...

Comment (vincentschen) [reply README.md]: 👍

**Pull request merged**